test/recipes/02-test_errstr.t

```diff
@@ -57,7 +57,18 @@ plan tests => scalar @Errno::EXPORT_OK
 foreach my $errname (@Errno::EXPORT_OK) {
     # The error names are perl constants, which are implemented as functions
     # returning the numeric value of that name.
-    &ok(match_syserr_reason("Errno::$errname"->()))
+    my $errcode = "Errno::$errname"->();
+
+    # On most systems, there is no E macro for errcode zero in <errno.h>,
+    # which means that it seldom comes up here.  However, reports indicate
+    # that some platforms do have an E macro for errcode zero.
+    # With perl, errcode zero is a bit special.  Perl consistently gives
+    # the empty string for that one, while the C strerror() may give back
+    # something else.  The easiest way to deal with that possible mismatch
+    # is to skip this errcode.
+    continue if $errcode == 0;
```

Comment:
This line is causing the number of tests executed to mismatch the expected number. So instead, the following could be used, replacing 69-71:
```
    if ($errcode == 0) {
        &ok(1) # Ignore this error code
    } else {
        &ok(match_syserr_reason($errcode))
    }
```

Reply:
Ahhh, thanks for catching that.  There's actually a well defined mechanism with the Test::More module to do skips, I'll employ that.

Reply:
Fixed.  Please try again

Reply:
Side note: just noticed what inspired you to have a `&` prefixing the function calls...  I do *not* know why some just have to add those, it's completely unnecessary except for some quite special circumstances...

Reply:
There was a comment in the test infrastructure about using `&ok` being required, so I went with it.

Reply:
[goes hide in a corner]

That was written by [ahem] me.  I had forgotten, and yes, what that comment says is true, in this particular file.  That's one of those special circumstances.